README.md

```diff
@@ -64,6 +64,12 @@ services:
 $ docker stack deploy --orchestrator=kubernetes -c docker-compose.yml hellokube
 ```
 
+## Remove a stack
+
+```
+$ docker stack deploy --orchestrator=Kubernetes hellokube
```

Comment:
Thank you @masayuki14 for your contribution 👍 But actually to remove a stack using the `docker cli` you should do something like this:

```sh
$ docker stack rm --orchestrator=kubernetes hellokube
```

Reply:
Oops. I had mistaken.
It's important to check myself beforehand.
I modified it.